src/features/utils/ansibleCfg.ts

```diff
@@ -0,0 +1,74 @@
+import * as vscode from 'vscode';
+import * as fs from 'fs';
+import untildify from 'untildify';
+import * as ini from 'ini';
+
+// Get rootPath based on multi-workspace API
+export function getRootPath(editorDocumentUri : vscode.Uri) {
+    let rootPath : string | undefined = undefined;
+
+    if (!!vscode.workspace.workspaceFolders) {
+        rootPath = vscode.workspace.workspaceFolders.length ? vscode.workspace.workspaceFolders[0].name : undefined;
+    }
+
+    if (!!vscode.workspace.getWorkspaceFolder) {
+        let workspaceFolder = vscode.workspace.getWorkspaceFolder(editorDocumentUri);
+
+        if (!!workspaceFolder) {
+            rootPath = workspaceFolder.uri.path;
+        } else {
+            // not under any workspace
+            rootPath = undefined;
+        }
+    }
+
+    return rootPath;
+}
+
+export function scanAnsibleCfg(rootPath : any = undefined) {
+    console.log(`util.scanAnsibleCfg(${rootPath})`);
```

Comment:
Is there a reason to leave the log statements in the code?

Reply:
not really, removed in 11ac5dd